packages/web-workers/src/orchestration-steps.ts

```diff
@@ -158,6 +160,23 @@ export class OrchestrationStepsImpl implements OrchestrationSteps {
         return scanId;
     }
 
+    public *invokeSubmitConsolidatedScanRequestRestApi(
+        url: string,
+        reportId: string,
+    ): Generator<Task, string, SerializableResponse & void> {
+        const requestData: CreateConsolidatedScanRequestData = {
+            scanUrl: url,
+            reportId: reportId,
+            priority: 1000,
+        };
+
+        const response = yield* this.callWebRequestActivity(ActivityAction.createConsolidatedScanRequest, requestData);
+        const scanId = yield* this.getScanIdFromResponse(response, ActivityAction.createConsolidatedScanRequest);
+        this.logOrchestrationStep(`Orchestrator submitted scan with scan Id: ${scanId}`);
```

Comment:
message can be extended like `Orchestrator submitted consolidated scan request with scan Id:

Reply:
changes show in most second most recent commit